requirements/common.txt

```diff
@@ -18,6 +18,8 @@ plotly==5.17.0
 pprintpp==0.4.0
 psutil>=5.7.0
 pymongo>=3.12
+pydantic==2.6.4
```

Comment:
Stuart had removed `pydantic` in the past. Is bringing back here intentional/needed?

Reply:
Including `pydantic` has longterm implications. Would be good to discuss offline

Reply:
Looks like it's not used anywhere
```suggestion
```